app/indexers/plum_jsonld_converter.rb

```diff
@@ -47,7 +47,8 @@ def output
       "date_created" => date_created,
       "description" => description,
       "series" => series,
-      "provenance" => provenance
+      "provenance" => provenance,
+      "collections_label" => collections
```

Comment:
Maybe call this `collection_labels` since it's a multivalue field

Reply:
I can't, the wildcard field is _label in the solr schema.

Reply:
bummer